[master regression 2022-09-26] `limactl start` segfaults

```console
$ limactl start
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xb0 pc=0x15cb2b3]

goroutine 1 [running]:
main.startAction(0xc0001c7b80, {0x1c5ce60, 0x0, 0x0})
        /Users/suda/gopath/src/github.com/lima-vm/lima/cmd/limactl/start.go:463 +0xf3
github.com/spf13/cobra.(*Command).execute(0xc0001c7b80, {0x1c5ce60, 0x0, 0x0})
        /Users/suda/gopath/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:872 +0x694
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001c7900)
        /Users/suda/gopath/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/suda/gopath/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
main.main()
        /Users/suda/gopath/src/github.com/lima-vm/lima/cmd/limactl/main.go:23 +0x1e
```

Regression in https://github.com/lima-vm/lima/commit/e5a79f13f048c8d1e1df84362eb43c4c1f668e63

CI was green though

--- a/cmd/limactl/start.go
+++ b/cmd/limactl/start.go
@@ -150,7 +150,8 @@ func loadOrCreateInstance(cmd *cobra.Command, args []string) (*store.Instance, e
 		if err := identifiers.Validate(st.instName); err != nil {
 			return nil, fmt.Errorf("argument must be either an instance name, a YAML file path, or a URL, got %q: %w", st.instName, err)
 		}
-		if inst, err := store.Inspect(st.instName); err == nil {
+		inst, err := store.Inspect(st.instName)
+		if err == nil {
 			logrus.Infof("Using the existing instance %q", st.instName)
 			if arg == "" {
 				logrus.Infof("Hint: To create another instance, run the following command: limactl start --name=NAME template://default")

